communication/nats/discovery/config.go

```diff
@@ -23,6 +23,6 @@ import "time"
 const (
 	BrokerPort          = 4222
 	BrokerMaxReconnect  = -1
-	BrokerReconnectWait = 4 * time.Second
+	BrokerReconnectWait = 1 * time.Second
```

Comment:
It would be nice all these tweaks to be configurable from cmd line, with sensible defaults

Reply:
The issue for that added: https://github.com/mysteriumnetwork/node/issues/818

Reply:
👍